testnets/jackaltestnet2/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "jackaltestnet",
+  "assets": [
+    {
+      "description": "The native staking and governance token of Jackal.",
+      "denom_units": [
+        {
+          "denom": "ujkl",
+          "exponent": 0
+        },
+        {
+          "denom": "jkl",
+          "exponent": 6
+        }
+      ],
+      "base": "ujkl",
+      "name": "Jackal",
+      "display": "jkl",
+      "symbol": "JKL",
```

Comment:
please define a `trace` to mainnet jackal using a trace "type": "test-mintage"

Reply:
I've never seen this before in the asset lists, do you have an example of another chain using this field?

Reply:
E.g., KYVE Testnet uses this correctly
https://github.com/cosmos/chain-registry/blob/78d62856452d99890e2486babd0c1abd831da934/testnets/kyvetestnet/assetlist.json#L23